### PR TITLE
feat: add cc genesys_eb_events and consumer and producers configs

### PR DESCRIPTION
**Pull request by vitorf7**

### CODE REVIEW

**Comment by vitorf7** [comment]
@utilitywarehouse/pubsub Does @tmus (as I probably would not have permissions to do so and I am assuming @tmus does) have to run `terraform init` in order to some sort of installation of the TLS module or anything like that? My LSP was telling me it was not installed but it could have just been a local thing.

File: dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf [hunk 1]
```diff
@@ -1,3 +1,17 @@
+resource "kafka_topic" "genesys_eb_events" {
+  name = "contact-channels.genesys_eb_events"
+
+  replication_factor = 3
+  partitions         = 9
+
+  config = {
+    "retention.ms"      = "2629800000" # 1 month
```
Comment (sbuliarca) [inline dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf:8]: Would it be ok if we enable tiered storage for this topic, and only keep, let's say 3 days into local storage?
Details and an example is at point 5 in https://github.com/utilitywarehouse/kafka-cluster-config/tree/main/dev-aws/kafka-shared-msk#contributing

Comment (vitorf7) [reply dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf:8]: @sbuliarca If I understand correctly if we enable that it means that the data will be available in `hot` storage (aka faster reading speeds) for 3 days and then moved to low-cost storage (slower reading speeds at least at the beginning of reading) up to the max of 1month as I have set here?

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf:8]: Exactly

**Comment by sbuliarca** [comment]
> @utilitywarehouse/pubsub Does @tmus (as I probably would not have permissions to do so and I am assuming @tmus does) have to run `terraform init` in order to some sort of installation of the TLS module or anything like that? My LSP was telling me it was not installed but it could have just been a local thing.

Nope. It will be applied automatically by https://terraform-applier-system.dev.aws.uw.systems/#pubsub-kafka-shared-msk-contact-channels. More details here: https://wiki.uw.systems/posts/how-to-migrate-to-amazon-msk-ztoacwsm#htsno-2-create-topics-in-msk

File: dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf (hunk 1 above)
Comment (vitorf7) [reply dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf]: @sbuliarca I have added the `local.retention.ms` option. Is that the only one? I saw some people had `remote.storage.enable` so I was not sure if that is needed as well.

Comment (sbuliarca) [reply dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf]: You need the `remote.storage.enable` too

Comment (vitorf7) [reply dev-aws/kafka-shared-msk/contact-channels/contact-channels.tf]: @sbuliarca Thank you. Added it now.

**Review by sbuliarca — APPROVED** [review]

**Pull request merged**